common/lib/xmodule/xmodule/capa_base.py

```diff
@@ -565,6 +565,21 @@ def handle_problem_html_error(self, err):
 
         return html
 
+    def _demand_hints_possible(self, demand_hints):
```

Comment:
Note that in Python, you can return 2 variables (and you can ignore the return value of one using __). So you could have introduced just a single helper method.

I imagine there will be a quality error here for no doc.